sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -41,6 +41,27 @@
       :title="studentMenu.applicationStatus.title"
       @click="studentMenu.applicationStatus.command"
     />
+    <v-list v-if="applicationHistory.length" density="compact" nav>
+      <v-list-group
+        v-for="application in applicationHistory"
+        :key="application.title"
+        :prepend-icon="application.props?.prependIcon"
+        :title="application.title"
+        :value="application.title"
+      >
+        <template v-slot:activator="{ props }">
```

Comment:
For slot, we use generally use the shortcut `#`. like this `<template #activator="{ props }">`